app/models/user.rb

```diff
@@ -53,10 +54,51 @@ class User < ActiveRecord::Base
             },
             presence: true
 
+  ##
+  # Checkes if the user attended the event
+  # This is used for events that require registration
+  # The user must have registered to attend the event
+  # Gets an event
+  # === Returns
+  # * +true+ if the user attended the event
+  # * +false+ if the user did not attend the event
+  def attended_event? event
+    event_registration = event.events_registrations.find_by(registration: self.registrations)
+
+    return false unless event_registration.present?
```

Comment:
You mean why I use `.present?` instead of `.any?` ? Or why I use a variable? (I use a variable because I am re-using it on the next line - thought it would look cleaner that way).